bands/templates/bands/technician_card.html

```diff
@@ -0,0 +1,15 @@
+{% if user in object.light_tech.all or user in object.sound_tech.all %}
```

Comment:
It'd be more efficient to filter in the view (`TechnicianList.get_queryset()`) or model (write a method `Concert.tech_responsibilities_for(user)` or similar).